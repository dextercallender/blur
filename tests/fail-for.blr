int main() {
    int i;
    for(i=0; "hey"; i=i+1) {
	println(i);
    }
}
