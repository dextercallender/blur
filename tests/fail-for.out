Fatal error: exception Failure("illogical for")
